ts/defs.bzl

```diff
@@ -152,6 +153,10 @@ def ts_project(
             See https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options
             Typically useful arguments for debugging are `--listFiles` and `--listEmittedFiles`.
 
+        isolated_declarations: Whether to enforce that declaration output (.d.ts file) can be produced for a
+            single source file at a time. Requires some additional explicit types on exported symbols.
```

Comment:
Is this "Equivalent to the TypeScript --isolatedDeclarations option." like the docs for other params say?